src/lib/modules/Items/backoffice/ItemListEntry.js

```diff
@@ -78,23 +79,57 @@ export class ItemListEntry extends Component {
             <Grid.Column computer={6} largeScreen={6}>
               <Item.Meta className="metadata-fields">
                 <Header as="h5">
-                  <LiteratureTitle
-                    title={item.metadata.document.title}
-                    truncateLines={1}
-                  />
+                  {checkIn ? (
+                    <Link
+                      target="_blank"
+                      to={BackOfficeRoutes.documentDetailsFor(
+                        item.metadata.document_pid
+                      )}
+                    >
+                      <LiteratureTitle
+                        title={item.metadata.document.title}
+                        truncateLines={1}
+                      />
+                    </Link>
+                  ) : (
+                    <LiteratureTitle
+                      title={item.metadata.document.title}
+                      truncateLines={1}
+                    />
+                  )}
                 </Header>
                 <List>
                   <List.Item>
-                    <List.Content>
-                      {item.metadata.internal_location.location.name} -{' '}
-                      {item.metadata.internal_location.name}
-                    </List.Content>
+                    {checkIn ? (
```

Comment:
you are on a good track.
having in mind the previous comment, I would change `checkin` prop name to `highlightLocation` and use it only for this purpose. It might come up handy and more flexible if we want to highlight the location in the other places this component is used.